src/components/Accordion/src/Accordion.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+	<div
+		:class="$s.Accordion"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div
+			:class="$s.AccordionHeader"
+			@click="toggleExpanded"
+		>
+			<div
+				:class="$s.TitleIconAligner"
```

Comment:
I tried to use the Row component initially but it didn't work for me because the icon/suffix slot is center-aligned between the title slot and the secondary slot, but for the Accordion designs the icon slot needs to be aligned with just the title slot and the secondary slot hangs below on its own.